mknotes.sh

```
@@ -0,0 +1,15 @@
+#쉘스크립트 용어
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)